lib/client/dom.js

```diff
@@ -883,6 +883,14 @@ var CloudCmd, Util, DOM, CloudFunc;
                 var current     = currentFile || this.getCurrentFile();
                     
                 current.classList.toggle(SELECTED_FILE);
+
+                // OSC_CUSTOM_CODE There is a bug where deselection is delayed, this is because the "selected-file"
+                // tag is successfully removed, but not the "current-file" tag. This removes the "current-file" tag
+                // if a file is deselected.
+                // https://github.com/AweSim-OSC/osc-fileexplorer/issues/59
+                if (current.classList.contains(CURRENT_FILE) && !current.classList.contains(SELECTED_FILE)){
+                    current.classList.remove(CURRENT_FILE);
+                }
```

Comment:
So in this case we could end up with one or more files selected where each selected file has the SELECTED_FILE class but not the CURRENT_FILE class. When opening a context menu does this make sense? Would there be a problem if we end up with a single selected file that is SELECTED_FILE and not CURRENT_FILE? Would there be a problem if there were 2 SELECTED_FILEs and no CURRENT_FILE?